source/extensions/filters/udp/udp_proxy/router/router.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "envoy/common/pure.h"
+#include "envoy/network/address.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace UdpFilters {
+namespace UdpProxy {
+namespace Router {
+
+/**
+ * The router.
```

Comment:
It would be nice to have a short high level explanation of what a Router does in this context. Presumably it looks as a source_address and returns a cluster name to send that packet to?

Reply:
Ok.